charts/tfy-jspolicy-configs/README.md

```diff
@@ -0,0 +1,22 @@
+# tfy-jspolicy-config helm chart packaged by TrueFoundry
+A Helm chart for jspolicy configurations
+
+## Parameters
+
+### replaceImageRegistry Configuration options for replacing the image registry
+
+| Name                                          | Description                                                                                                     | Value   |
+| --------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | ------- |
+| `replaceImageRegistry.enabled`                | Enable or disable replacing the image registry                                                                  | `false` |
+| `replaceImageRegistry.excludeNamespaces`      | Namespaces to exclude from replacing the image registry                                                         | `[]`    |
+| `replaceImageRegistry.includeNamespaces`      | Namespaces to include from replacing the image registry. When non-empty, only these namespaces will be included | `[]`    |
```

Comment:
Which one trumps? What if I put the same namespace in include and exclude? Can you add docs for this?

Reply:
It says this - `When non-empty, only these namespaces will be included`